src/Form/ResearchGroupType.php

```diff
@@ -33,6 +61,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('id', TextType::class, array(
                 'label' => 'Research Group ID:',
+                'attr' => array('placeholder' => $this->researchGroupRepository->getNextAvailableId(self::MIN_ID, self::MAX_ID)),
```

Comment:
This is ok with me, but check the PO. And you then may want to prefix it with: "next avail ID is...."

Reply:
Believe me, there is no other way than to use placeholder without getting into a major mess with that current multi-modal form.

Reply:
It is doable, but it's messy.

Look at the code below, you can find the attribute here, and set it. But it would also cause issues while editing, and this field should locked then.

https://github.com/griidc/pelagos/blob/develop/assets/static/js/entity/ResearchGroup.js#L39-L42